directories/src/redux/reducers/node-reducer.js

```diff
@@ -24,16 +24,13 @@ const brokerNodeGenerator = address => {
 const newGenesisHashGenerator = (genesisHash, numberOfChunks) => {
   const numberOfSectors = Math.ceil(numberOfChunks / CHUNKS_PER_SECTOR);
   const sectorIndexes = _.range(numberOfSectors);
-  const sectors = _.chain(sectorIndexes)
-    .map(index => {
-      return {
-        index,
-        status: SECTOR_STATUS.NOT_STARTED
-      };
-    })
-    .shuffle()
```

Comment:
not going to randomize the order of new genesis hashes (per https://docs.google.com/document/d/1JaSKWDbPJQNLlAJyt4aNgJcGGS3gcv7Gz7AMeYG0EWI/edit#)